services/QuillLMS/client/app/bundles/Teacher/components/teacher_guide/getting_started_mini.jsx

```diff
@@ -1,38 +1,43 @@
 import React from 'react'
+
 import PercentageGraph from './percentage_graph'
 import CheckboxSection from './checkbox_sections'
+
 import LoadingIndicator from '../shared/loading_indicator'
 
 export default class GettingStartedMini extends React.Component {
   checklistSection() {
-    if (this.props.checkboxData.loading) {
+    const props = this.props
+    if (props.checkboxData.loading) {
```

Comment:
I would use deconstruction (`const { checkBoxData, } = this.props`) here rather than just saving the `props` variable (same for the other instances in this file).